.github/workflows/deploy-api.yml

```diff
@@ -11,6 +11,11 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      # heroku only uses yarn if there's a yarn.lock file
+      - name: Use Yarn
+        run: touch yarn.lock
```

Comment:
another option - `cp ../yarn.lock yarn.lock`

Reply:
(this is probs safer for matching versions of packages, but not a huge deal)

Reply:
love it